dataprofiler/labelers/data_processing.py

```diff
@@ -107,40 +116,35 @@ def get_parameters(self, param_list=None):
                 )
         return param_dict
 
-    def set_params(self, **kwargs):
+    def set_params(self, **kwargs: Any) -> None:
         """Set the parameters if they exist given kwargs."""
         # first check if any parameters are invalid
         self._validate_parameters(kwargs)
 
         for param in kwargs:
             self._parameters[param] = kwargs[param]
 
-    @abc.abstractmethod
-    def process(self, *args):
```

Comment:
`mypy` creates a errors for `process` in each subclass because the type signature is different, so I removed it here

Reply:
hmm what is you just remove `@abc.abstractmethod` as the decorator. We would still want the `def process()` in the base class definition so it enforces that function in all child classes.

Reply:
@tonywu315, I'd prefer to keep this, but if its causing an issue with `mypy` glad to talk about that. CC @JGSweets

Reply:
per @tonywu315, `Signature of "process" incompatible with supertype "BaseDataProcessor"` is how `mypy` complains about the `def process()`

Reply:
I would be possible to keep it if we added default values to `results` and `label_mapping` for all of the postprocessors, which would make them not violate Liskov substitution principle.